.github/workflows/continuous-integration.yaml

```diff
@@ -728,47 +736,8 @@ jobs:
         run: |
           make pytest PYTEST_OPTIONS=--weekly
 
-      # Run Pytest on codeblocks if the source code or any markdown has changed, or if this is
-      # part of the weekly or release CI. This step is also triggered if any dependency has been 
-      # updated or if some changes were found in the Makefile
-      - name: PyTest CodeBlocks
```

Comment:
(just moved below for clarity : basically I grouped pytests steps and codeblocks steps together to better understand what's happening here)